macros/src/lib.rs

```diff
@@ -30,26 +30,54 @@ struct DerivedTS {
 
 impl DerivedTS {
     fn into_impl(mut self, rust_ty: Ident, generics: Generics) -> TokenStream {
-        let mut get_export_to = quote! {};
-        let export_to = match &self.export_to {
-            Some(dirname) if dirname.ends_with('/') => {
-                format!("{}{}.ts", dirname, self.ts_name)
-            }
-            Some(filename) => filename.clone(),
-            None => {
-                get_export_to = quote! {
-                    fn get_export_to() -> Option<String> {
-                        ts_rs::__private::get_export_to_path::<Self>()
-                    }
-                };
-                format!("bindings/{}.ts", self.ts_name)
-            }
-        };
-
         let export = self
             .export
             .then(|| self.generate_export_test(&rust_ty, &generics));
 
+        let export_dir = std::env::var("TS_RS_EXPORT_DIR").ok();
```

Comment:
Ahh, I think this here might be the issue!  
This is evaluated when the macro executes, during compilation.  

When later running this executable, the `TS_RS_EXPORT_DIR` is hard-coded in there.  
That might also happen when just doing `cargo test`, since rust might not re-compile.

Reply:
I think you're right, I just commited a version using `std::option_env!` instead and that seems to have fixed it

Reply:
Still, it might be better to move reading the env variable to `export.rs` instead, because I think the way I just did it forces recompilation, which kinda sucks

Reply:
Agreed, both with the observation as well as the conclusion.

Reply:
The last change did cause a recompilation, but recompilation of `ts-rs-macros`.  
The 3 possible behaviours we could get are
- Re-compile ts-rs-macros 
  Use `env!` in `ts-rs-macros`
- Re-compile the consumer crate
  Load the environment variable in `export.rs`, using `env!()`
- Don't recompile anything
  Load the environment variable in `export.rs`, using `env::var()`

Reply:
Got it fixed! Sorry about the billion failed CI notifications 😅

Reply:
> * Don't recompile anything
>       Load the environment variable in `export.rs`, using `env::var()`

Ended up on this one, my cpu is already thanking me 😆